README.md

```diff
@@ -74,3 +74,47 @@ fn main() {
     }
 }
 ```
+
+Testing
+-------
+
+The driver test suite is largely composed of a mixture of integration tests and behavioral unit-tests, relying on the official [MongoDB specifications repo](https://github.com/mongodb/specifications). The easiest way to thoroughly test the driver is to set your fork up with TravisCI.
+
+To test the driver locally, you'll need to setup integration tests and specification tests.
+
+> NOTE: Each integration test uses a unique database/collection to allow tests to be parallelized, and will drop their dependencies before running. However, effects are _not_ cleaned up afterwards.
+
+#### Setting up integration tests
+
+All integration tests run on the default MongoDB port, 27017. Before running the tests, ensure that you setup a test database on that port.
+
+Download a version of MongoDB:
+```
+wget http://fastdl.mongodb.org/linux/mongodb-linux-x86_64-3.2.11.tgz
```

Comment:
It's probably better to just point users to [https://www.mongodb.com/download-center](https://www.mongodb.com/download-center) so they can download whichever release they need rather than specifying the Linux link directly.